### ISSUE TITLE
Message text for author request email

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTDESC/desc-publications#70]
Here's an attempt at what the message from the pub system to the primary authors should say, when someone requests authorship via the pub system UI. What do you think, @LSSTDESC/pubboard ?

> Subject: Request for authorship on your publication "**pub-title**" (**pub-id**) from <user-full-name> 
>
> Dear Primary Authors,
>
> **user-full-name** is asking to be considered as a co-author of your publication "**pub-title**" (**pub-id**). You can read their justification below. When you have converged on a good response, please reply to them via the publication mgmt system at [this link](**response-url**), and update the author list as required. For guidance on authorship criteria, please consult [the LSST DESC publication policy](**pub-policy-url**), and if in doubt, please don't hesitate to contact the LSST DESC pub board (cc'd on this email). 
>
> Thanks!
>
> The DESC Publication Management System
>
> **user-full-name** writes:
> Reason for co-authorship: **reason-text** 
> Justification: **justification-text**

Apologies for the pseudocode - I don't actually know the correct variable names but hopefully the strings in bold above make sense.

Here's the current message, from a recent test by @chee999 : 
> Reason: Test out the request for authorship feature.
> If this were a real request I would have to give a convincing reason why I should be an author on this paper.  Selected Contributions: Conceptualization, Formal Analysis, Investigation, Project Administration, Software, Validation, Writing - Original Draft

**Comment by rmandelb** [comment]
I agree that the current message is insufficient.  Your draft looks good to me.  But... will the PB really be cc'ed on every single authorship request for every single paper?  No big deal right now, but when things get going that could be a bit nuts.  I feel like it would be better to avoid that, in favor of some kind of periodic big-picture summary (i.e., an update at some sensible interval that publication 374 has gotten 17 authorship requests, of which 14 were accepted and 3 declined, and an option to get the details of the requests and the decision rationale if needed).  I know that if I'm getting many e-mails about authorship requests for many papers at a time, I will lose any sort of big picture and would prefer some kind of periodic status update.

**Comment by drphilmarshall** [comment]
Good point. Our opt-in system will generate a _lot_ of email traffic. Does the Pub Manager even need to be cc'd? Maybe not. My logic was to try to make it as easy as possible for the authors to get in touch wit the pub board, but we could just put a mailto link in instead.

**Comment by abmantz** [comment]
I completely agree that the PB doesn't need to get email about each author request.

I do have a general request about emails that the system will send (not just this case): the name of the publication should be the very first thing in the body of the email, by itself, ideally in giant, bold type. As in

# Re: DESC-NNNN  "Name of the publication"
```
Additional info
```

In contrast, the DES system tends to head things with paragraphs like
```
DES publication:        [DES-NNNN] Title
DES Project:            [NNNN] Name
Working Groups:         A, B, C
Principal author(s):    Enormously long list of names. Seriously, like 5 lines worth
Review committee:       Even more names

Publication state has changed microscopically
```
That's in the best case, where there isn't a "Dear collaborators, below is an annoying message" piece at the top. It often takes me 5-10 seconds to decipher these and conclude that I don't care about these message, and I'm not even particularly dyslexic.

Sure, I'm complaining about DES here. But the point is that, eventually, members will being seeing a huge amount of publication-related spam, and being able to instantly filter out the majority that they don't care about is valuable.

**Comment by chee999** [comment]
The email is intended to go to the primary authors only.  If and only if the primary authors have an issue with approving then they are suppose to contact the publication board.  Currently, the requests are emailed and then deleted from the database. From this discussion it appears the requests should be kept along with how many times a document receives a request and the outcome of the request.

**Comment by rmandelb** [comment]
@chee999 - having the e-mail go to the primary authors only makes sense to me.  But I agree the requests should be kept along with outcomes.  There are sometimes going to be cases where somebody wants to appeal, or where the PB might notice some anomalous pattern they want to investigate (27 requests, 0 approved - perhaps something unreasonable is happening?), so having a record would be helpful.

**Comment by abmantz** [comment]
What exactly is the expected workflow here?
1. Authors receive a request
2. To accept it, authors have to log in and add the requesting author themselves, cut and pasting the contribution statement? Click an "accept" button, with the requested contribution statement going in by default? Could it instead be the case that authors only need to take action to _not_ accept a request?

**Comment by sethdigel** [comment]
>What exactly is the expected workflow here?
>
> 1.   Authors receive a request
> 2.   To accept it, authors have to log in and add the requesting author themselves, cut and pasting the contribution statement? Click an "accept" button, with the requested contribution statement going in by default? Could it instead be the case that authors only need to take action to not accept a request?

Seems simple enough  - only two steps! 

Yes, step 2 does look like it has potential to be error prone.  I think we should deal with that in a separate issue, which I'll belatedly define separately.

Regarding the contribution statements that are entered at this stage (see  #82), I think that they are the statements that the Pub Policy says will be made available online.  We need to make clear that the concise statements that go in the acknowledgments (for Primary authors only) will  need to be composed separately.   These concise statements really cannot be entered at this stage because in the  paper they will often get combined (e.g., instead of 'ABM developed analysis methods.  PJM developed analysis methods.' a paper would have 'ABM and PJM developed analysis methods.'

**Comment by sethdigel** [comment]
Regarding the text of the message (the subject of this issue), here's what I'd suggest, on the assumption that we can handle author list management separately.  Note that I have removed 'reason-text' and 'justification-text', on the assumption that they would overlap each other and also overlap  'contribution-text'.  I've also dropped 'Thanks!' because it is creepy for an automated system to thank someone.

  Subject: Request for authorship on your publication "pub-title" (pub-id) from user-full-name

**Re:  DESC-NNNN "pub-title"**
Dear Primary Authors,

  user-full-name is asking to be considered as a co-author of your publication and has provided the justification below.  Please accept or decline at [this link]((http://something).  For guidance on authorship criteria, please consult the DESC Publication Policy, and if in doubt, please don't hesitate to contact the [DESC Pub Board](mailto:lsst-desc-publications@slac.stanford.edu).

The DESC Publication Management System

    user-full-name
    Proposed contribution statement: contribution-text

**Comment by sethdigel** [comment]
(Documenting a discussion with Charlotte).  The e-mail message will omit the contribution-text.  The message will read

Re: DESC-NNNN "pub-title"
Dear Primary Authors,

user-full-name is asking to be considered as a co-author of your publication.  Please accept or decline at [this link]((http://something), which also includes the contribution statement that was provided. For guidance on authorship criteria, please consult the DESC Publication Policy, and if in doubt, please don't hesitate to contact the [DESC Pub Board]((lsst-desc-publications@slac.stanford.edu).

The DESC Publication Management System

**Comment by drphilmarshall** [comment]
Good idea to drive the PA's back to the web interface.

**Comment by sethdigel** [comment]
Charlotte has implemented the authorship request e-mail messages with the template that  we asked for.
